import/popups/DownloadMenuItem.qml

```diff
@@ -12,6 +12,7 @@
 import QtQuick 2.0
 import Sailfish.Silica 1.0
 import Sailfish.WebEngine 1.0
+import Sailfish.Pickers 1.0
 
 MenuItem {
     id: root
```

Comment:
The translation hints in ContextMenu.qml for "Save link" and "Save image" no longer apply, so should probably be updated with this PR too:
```
//: This menu item saves link to downloads.
//: This menu item saves image to Gallery application
```